.changeset/sixty-dogs-sneeze.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": minor
+---
+
+Adds an experimental option to prefetch to prerender pages on the client with the Speculation Rules API when supported.
```

Comment:
``````suggestion
Adds an experimental flag `clientPrerender` to pre-render your prefetched pages on the client with the [Speculation Rules API](https://developer.mozilla.org/en-US/docs/Web/API/Speculation_Rules_API).

```js
// astro.config.mjs
{
  prefetch: {
    prefetchAll: true,
    defaultStrategy: viewport,
  }
  experimental: {
    clientPrerender: true,
  },
}
 ```

Enabling this feature overrides the default `prefetch` behavior globally to pre-render all links with the `data-astro-prefetch` attribute on the client. Instead of appending a `<link>` tag to the head of the document or fetching the page with JavaScript, a `<script>` tag will be appended with the corresponding speculation rules.

Client side pre-rendering requires browser support. If the Speculation Rules API is not supported, `prefetch` will fallback to the supported strategy.

See the [Prefetch Guide](https://docs.astro.build/en/guides/prefetch/) for more `prefetch` options and usage.
``````
Check that this is accurate, but I think for a minor, this is helpful context here in the changeset/for the blog post!